variables.tf

```diff
@@ -69,6 +69,24 @@ variable "cpu" {
   type        = number
 }
 
+variable "code_pipeline_artifact_bucket" {
+  default     = ""
+  description = "Use an existing bucket for codepipeline artifacts that can be reused for multiple services."
+  type        = string
+}
+
+variable "code_pipeline_role" {
```

Comment:
shall we call this `code_pipeline_role_name` to clarify that we need the name and not the arn?